src/entrypoints/popup/Chat.svelte

```diff
@@ -1,18 +1,36 @@
 <script lang="ts">
   import Wrapper from "./Wrapper.svelte";
   import { Send } from "lucide-svelte";
-
   var text = $state<string>();
 </script>
 
 <Wrapper pageTitle="Ducky Chat">
-  <div class="response-container">
-    <p>Ask me a question!</p>
-  </div>
+  <p id="response">Ask me a question!</p>
   <form
     class="form"
-    onsubmit={() => {
-      alert(text ?? "");
+    onsubmit={(event) => {
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading."; }, 250);
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading.."; }, 500);
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading..."; }, 750);
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading.."; }, 1000);
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading."; }, 1250);
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading.."; }, 1500);
+      // prettier-ignore
+      setTimeout(() => { document.getElementById("response")!.innerText = "Loading..."; }, 1750);
```

Comment:
Bruh. Ever heard of [keyframes](https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes)?